tools/reviewer/local_server/service/tests/BUILD

```diff
@@ -39,15 +34,11 @@ java_test(
         "//common/flags",
         "//common/repo",
         "//common/repo:repo_java_proto",
-        "//third_party/maven/com/google/flogger",
         "//third_party/maven/com/google/flogger:flogger_system_backend",
-        "//third_party/maven/com/google/guava",
         "//third_party/maven/io/grpc:grpc_core",
-        "//third_party/maven/io/grpc:grpc_netty",
-        "//third_party/maven/io/grpc:grpc_services",
+        "//third_party/maven/javax/inject:javax_inject",
```

Comment:
Why do we need `inject:javax_inject` and before we didn't?

Reply:
Actually, I don't know. 
`CodeReviewServiceGetDiffFilesTest` class contains imports:
- import javax.inject.Named;
- import javax.inject.Singleton;
But the test successfully builds and runs without `//third_party/maven/javax/inject:javax_inject`. Maybe some other dependency comtains `import javax.inject.Named` and `import javax.inject.Singleton` inside.